creator-node/src/services/stateMachineManager/stateMonitoring/findReplicaSetUpdates.jobProcessor.js

```diff
@@ -54,9 +54,13 @@ module.exports = async function ({
           user,
           thisContentNodeEndpoint,
           unhealthyPeersSet,
-          userSecondarySyncMetricsMap[user.wallet],
+          userSecondarySyncMetricsMap[user.wallet] || {
+            [user.secondary1]: { successRate: 1, failureCount: 0 },
+            [user.secondary2]: { successRate: 1, failureCount: 0 }
+          },
```

Comment:
what's the intention behind the default for `successRate: 1`?

Reply:
it's because we should assume it's safe to sync if there were no failures logged (which is default -- no data yet). we want it to have a high enough default success rate or else it will fail to sync when it's probably just fine to sync

Reply:
I feel like this is conflating code intentions and implementation. like if no syncs has been performed, we should just sync VS marking a user with successful sync just to trigger a sync. I think this is ok for now

Reply:
oh no sorry that's not what I meant -- it's not to trigger a sync. it's more that if the failure rate is low then that prevents a sync, so we don't want to prevent a necessary sync. it'll still only sync if a sync is actually needed (i.e. the clock value is behind and no other failure conditions are met)